qiskit/quantum_info/states/utils.py

```diff
@@ -140,28 +140,27 @@ def schmidt_decomposition(state, qargs):
     orthonormal set of vectors in their respective spaces :math:`A` and :math:`B`,
     and the Schmidt coefficients :math:`\lambda_k` are positive real values.
 
+    ..note ::
+        In Qiskit, qubits are ordered using little endian notation, with the least significant
+        qubits having smaller indices. For example, a four-qubit system is represented as
+        :math:`|q_3q_2q_1q_0\rangle`. Using this convention, setting `qargs = [0]` will partition the
+        state as :math:`|q_3q_2q_1\rangle_A\otimes|q_0\rangle_B`. Furthermore, qubits will be organized
+        in this notation regardless of the order they are passed. For instance, passing either
+        `qargs = [1,2]` or `qargs = [2,1]` will result in partitioning the state as
+        :math:`|q_3q_0\rangle_A\otimes|q_2q_1\rangle_B`.
```

Comment:
The main problem with this is the `..note` rather than `.. note` (space) more than the location.  For completeness' sake, though:
```suggestion
    .. note::
        In Qiskit, qubits are ordered using little-endian notation, with the least significant
        qubits having smaller indices. For example, a four-qubit system is represented as
        :math:`|q_3q_2q_1q_0\rangle`. Using this convention, setting ``qargs=[0]`` will partition the
        state as :math:`|q_3q_2q_1\rangle_A\otimes|q_0\rangle_B`. Furthermore, qubits will be organized
        in this notation regardless of the order they are passed. For instance, passing either
        ``qargs=[1,2]`` or ``qargs=[2,1]`` will result in partitioning the state as
        :math:`|q_3q_0\rangle_A\otimes|q_2q_1\rangle_B`.
```

(fwiw, I think the text about endianness conventions confuses tensor-product order notation with the realisation of the Kronecker product, but that problem's endemic to our documentation [and some would consider to be me nitpicking], so doesn't need to be changed here)